samples/nrf9160/modem_shell/Kconfig

```diff
@@ -84,6 +84,13 @@ config MOSH_REST
 	help
 	  REST client for sending and receiving simple REST requests/responses.
 
+config MOSH_CLOUD
+	bool "MQTT connection to nRFCloud"
+	default n
```

Comment:
There's no need for "default n", because that's the default value. Somebody will anyway ask you to remove this line, so it may as well be me. :)